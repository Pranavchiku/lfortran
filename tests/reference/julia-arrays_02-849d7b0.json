{
    "basename": "julia-arrays_02-849d7b0",
    "cmd": "lfortran --no-color --show-julia {infile}",
    "infile": "tests/../integration_tests/arrays_02.f90",
    "infile_hash": "b5b3a6833fd727f5cce92c3c4886c4e20117f33710a1ec8b1016d8f6",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "julia-arrays_02-849d7b0.stdout",
    "stdout_hash": "bd215e16c4ff930ada33791182bddfd8bd27c401b70ed09c1b31facb",
    "stderr": "julia-arrays_02-849d7b0.stderr",
    "stderr_hash": "f08ef43d34c0650fb5ab3d37fca9c94ee0048dfde6f97d43c25c1534",
    "returncode": 0
}
