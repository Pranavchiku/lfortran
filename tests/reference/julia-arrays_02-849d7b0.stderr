warning: Implied save feature
 --> tests/../integration_tests/arrays_02.f90:3:1
  |
3 | integer, parameter :: dp = kind(0.d0)
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/../integration_tests/arrays_02.f90:4:1
  |
4 | real(dp) :: a(3), b
  | ^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/../integration_tests/arrays_02.f90:4:1
  |
4 | real(dp) :: a(3), b
  | ^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables
